tests/python/ci/test_mergebot.py

```diff
@@ -165,7 +158,8 @@ def test_mergebot(tmpdir_factory, number, filename, expected, comment, user, det
     }
     collaborators = []
 
-    proc = subprocess.run(
+    # return code is checked manually, so disable pylint
+    proc = subprocess.run(  # pylint: disable=subprocess-run-check
```

Comment:
We can use `check=False` here as well I think? I ran the tests and linter locally to check everything still works.